libass/ass.h

```diff
@@ -734,6 +734,24 @@ void ass_process_chunk(ASS_Track *track, const char *data, int size,
  */
 void ass_set_check_readorder(ASS_Track *track, int check_readorder);
 
+/**
+ * \brief Garbage collect events that preceed deadline
+ * \param track track
+ * \param deadline cut-off timestamp in milliseconds.
+*/
+void ass_prune_events(ASS_Track *track, long long deadline);
```

Comment:
SInce this adds new interfaces the `LIBASS_VERSION` macro in `ass.h` needs to be incremented on the second-least-significant nibble (the least-significant one is reserved for hotfixes) and the libtool version in `libass/Makefile_library.am` also needs to be bumped by zeroing `revision` and bumping both `current` and `age`